src/Tools/Md2Html/Md2Html.csproj

```diff
@@ -1,61 +1,48 @@
-		NDesk.Options.dll ^
-		System.Buffers.dll ^
-		System.Memory.dll ^
-		System.Numerics.Vectors.dll ^
-		System.Runtime.CompilerServices.Unsafe.dll ^
-		System.Text.Encoding.CodePages.dll " />
-    <Exec Condition=" '!$([MSBuild]::IsOSPlatform(`Windows`))' == 'True' "
-		Command="mkdir -p $(SharedOutputPath)"/>
-    <Exec Condition=" '!$([MSBuild]::IsOSPlatform(`Windows`))' == 'True' "
-		Command="cp $(OutputPath)*.* $(SharedOutputPath)"/>
-  </Target>
+<Project Sdk="Microsoft.NET.Sdk">
+    <ImportGroup Label="PropertySheets">
+        <Import Project="$(SolutionDir)Config\CS_SDK.props" />
+    </ImportGroup>
+    <PropertyGroup>
+        <ProjectGuid>{0893F745-CB1A-427A-8E87-CA927273039A}</ProjectGuid>
+        <OutputType>Exe</OutputType>
+        <RootNamespace>Md2Html</RootNamespace>
+        <AssemblyName>Md2Html</AssemblyName>
+        <SharedOutputPath>$(OutputPath)\Md2Html\</SharedOutputPath>
+        <OutputPath>bin\AnyCPU\$(Configuration)\</OutputPath>
+        <PublishSingleFile>true</PublishSingleFile>
+        <SelfContained>true</SelfContained>
+    </PropertyGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net48'">
+        <Reference Include="Microsoft.CSharp" />
+        <Reference Include="System.Data" />
+        <Reference Include="System.Net.Http" />
+        <Reference Include="System.Xml" />
+        <Reference Include="System" />
+        <Reference Include="System.Core" />
+        <Reference Include="System.Xml.Linq" />
+        <Reference Include="System.Data.DataSetExtensions" />
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="AngleSharp" Version="0.14.0" />
+        <PackageReference Include="AngleSharp.Css" Version="0.14.2" />
+        <PackageReference Include="CommandLineParser">
+            <Version>2.8.0</Version>
+        </PackageReference> 
+        <PackageReference Include="HtmlSanitizer">
+            <Version>5.0.372</Version>
+        </PackageReference>
+        <PackageReference Include="Markdig">
+            <Version>0.22.0</Version>
+        </PackageReference>
+        <PackageReference Include="System.Buffers" Version="4.5.1" />
+        <PackageReference Include="System.Memory" Version="4.5.5" />
+        <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />
+        <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" />
+        <PackageReference Include="System.Text.Encoding.CodePages" Version="7.0.0" />
+    </ItemGroup>
+    <Target Name="AfterBuildOps" AfterTargets="Build">
+        <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'" Command="if not exist $(SharedOutputPath) mkdir $(SharedOutputPath)" />
+        <Exec Condition=" '!$([MSBuild]::IsOSPlatform(`Windows`))' == 'True' " Command="mkdir -p $(SharedOutputPath)" />
+        <Exec Condition=" '!$([MSBuild]::IsOSPlatform(`Windows`))' == 'True' " Command="cp $(OutputPath)*.* $(SharedOutputPath)" />
+    </Target>
 </Project>
```

Comment:
we need this to run on net48 too. Please verify if the new way works for net48. If not you will have to bring back the old way with (ilmerge) under the net48 condition.
1. Net6 : SelfContained + PublishSingleFile
2. net48 - ilmerge (old way)

Reply:
@pinzart90 As I told you before, this was built it correctly on .NET 4.8 and also I built it on .NET 6, and there wasn't any error.

Reply:
@JimmySanchezGlobant please see the comment above from the bin diff job - the exe is not being produced:
https://github.com/DynamoDS/Dynamo/pull/14106#issuecomment-1613312262, so even if there are no errors, something seems off.

Reply:
@JimmySanchezGlobant the new flags do not work for net48 target.
You need to use the old way (with ILMerge) for net48
And the new flags + Add a new publish target (the flags work on publish only I think)
ex
```
<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Build;Publish">
  
```
But this should be added only under net6 target somehow. Also `SelfContained` should be set to false

Reply:
@pinzart90 The old way with ILMerge is failing, when I build the project it's returning the error code (missing file), so, it's not clear to me how this can solve this issue.

Reply:
@pinzart90 Yes, it's working well under . Net 4.8 from the master branch, it's weird, yesterday I did exactly the same test and it was failing on the ILMerge command, so, I don't know what to do to solve this, to be honest, I'm still very confused.